source/includes/connect/connection-options.cpp

```diff
@@ -0,0 +1,27 @@
+// start-connection-uri
+#include <mongocxx/uri.hpp>
```

Comment:
Suggest including a `main` in the examples so they can be copy-pasted and compiled without modification:

```c++
// start-connection-uri
#include <mongocxx/uri.hpp>
#include <mongocxx/client.hpp>

int main()
{
    mongocxx::uri uri("mongodb://<hostname>:<port>/?tls=true&tlsCertificateKeyFile=path/to/file.pem");
    mongocxx::client client(uri);
}
// end-connection-uri
```

Reply:
Added main method + instance object to all code examples